static-assets/components/cstudio-forms/controls/rte-tinymce5.js

```diff
@@ -676,14 +676,31 @@ CStudioAuthoring.Module.requireModule(
           return inputEl;
         },
 
+        /**
+         * Check input with space only
+         */
+        _isInputSpace: function (obj) {
+          if (!obj.validateTextArea) {
+            obj.validateTextArea = document.createElement('textarea');
+          }
+          obj.validateTextArea.innerHTML = obj.value;
+
+          if (!obj.validateDiv) {
+            obj.validateDiv = document.createElement('div');
+          }
+          obj.validateDiv.innerHTML = obj.validateTextArea.value;
+
+          return obj.validateDiv.innerText.trim() == '';
+        },
```

Comment:
Let's move to a common place so we don't duplicate code. I believe common api is common in the runtime `CStudioAuthoring.Utils.isEmptyHtml`
```suggestion
        isEmptyHtml: function (html) {
          const textarea = document.createElement('textarea');
          const div = document.createElement('div');
          textarea.innerHTML = html;
          div.innerHTML = textarea.value;
          return div.innerText.trim() == '';
        },
```